network/check.py

```diff
@@ -253,9 +249,35 @@ def _submit_regexed_values(self, output, regex_list):
                 if value:
                     self.rate(metric, self._parse_value(value.group(1)))
 
+    def _is_collect_cx_state_runnable(self, proc_location):
+        """
+        Determine if collect_connection_state is set and can effectively run.
+        If self._collect_cx_state is True and a custom proc_location is provided, the system cannot
+         run `ss` or `netstat` over a custom proc_location
+        :param proc_location: str
+        :return: bool
+        """
+        if self._collect_cx_state is False:
+            return False
+
+        if proc_location != "/proc":
```

Comment:
I'm not sure if it's interesting to add a condition over `Platform.is_containerized()`

Reply:
I think that's not necessary indeed